src/ext-test/java/org/opentripplanner/ext/flex/trip/ScheduledDeviatedTripTest.java

```diff
@@ -144,9 +144,9 @@ void calculateDirectFare() {
       List.of(to)
     );
 
-    var filter = new FaresFilter(graph.getFareService());
+    var filter = new DecorateWithFare(graph.getFareService());
 
-    var itineraries = filter.filter(router.createFlexOnlyItineraries().stream().toList());
+    var itineraries = router.createFlexOnlyItineraries().stream().peek(filter::decorate).toList();
```

Comment:
What is the reason to use peek rather than map?

Reply:
The decorate method return `void` (by design) - enforcing the client to change the itinerary, rather than returning a new one - this has some downsides, but this is how it works right now.